cucumber/authenticators_jwt/features/authn_jwt_fetch_signing_key.feature

```diff
@@ -740,7 +740,7 @@ Feature: JWT Authenticator - Fetch signing key
     And I successfully set authn-jwt "issuer" variable to value "valid-issuer"
     When I GET "/authn-jwt/raw/cucumber/status"
     Then the HTTP response status code is 500
-    And the authenticator status check fails with error "CONJ00120E Failed to parse 'public-keys': Type can't be blank, Type '' is not a valid public-keys type. Valid types are: jwks, and Value can't be blank"
+    And the authenticator status check fails with error "CONJ00120E Failed to parse 'public-keys': Type can't be blank, Value can't be blank, and Type '' is not a valid public-keys type. Valid types are: jwks"
```

Comment:
Looks like `Type can't be blank` appears twice in this change.

Reply:
You mean `Type can't be blank` and `Value can't be blank` are duplicates?

This error is auto-generated and concatenates several errors (as before).
In ruby 3 the order changed a bit

Reply:
Currently will keep this like that since only the order changed (and it auto-generated).
Resolving the conversation
Thanks